### PR TITLE
[Feature] Support when doppler project is different from repo name

### PR DESCRIPTION
**Pull request by sodafountain**
In case of monorepos, multiple projects within same repo will have different dopplre project names

### CODE REVIEW

File: .github/workflows/cd-common.yaml
Comment (fountain-draloff) [inline .github/workflows/cd-common.yaml:134]: Seems like we don't need this post-merge

Comment (sodafountain) [reply .github/workflows/cd-common.yaml:134]: Yeah I added that debug [commit](https://github.com/onboardiq/gh-common-workflows/pull/59/commits/b6ea9ba7c720125839986dd4719fd5119afda7c3) to unblock nestjs-backend from using this code. That plan was to remove the commit before merging.

**Review by fountain-draloff — APPROVED** [review]

**Pull request merged**
